api/_sql/common/V1.4.0__rfc_products.sql

```diff
@@ -0,0 +1,31 @@
+-- acquirable
+INSERT INTO acquirable (id, name, slug) VALUES
+    ('6c879d18-2eca-4b35-9fab-2b5f78262fa6', 'ncrfc-rtmat-01h', 'ncrfc-rtmat-01h'),
+    ('28d16afe-2834-4d2c-9df2-fdf2c40e510f', 'ncrfc-fmat-01h', 'ncrfc-fmat-01h'),
+    ('a483aa42-4388-4289-a41e-6b78998066a7', 'ncep-mrms-v12-msqpe01h-p2-carib', 'ncep-mrms-v12-msqpe01h-p2-carib'),
+    ('e5dfeef2-f070-49dc-8f3c-1c9230000f96', 'ncep-mrms-v12-msqpe01h-p1-carib', 'ncep-mrms-v12-msqpe01h-p1-carib'),
+    ('1860dfa9-0d2c-4b75-84ed-516792d940ee', 'ncep-mrms-v12-msqpe01h-p2-alaska', 'ncep-mrms-v12-msqpe01h-p2-alaska'),
+    ('cf75d07d-d527-4be0-b066-0bfa86565ab5', 'ncep-mrms-v12-msqpe01h-p1-alaska', 'ncep-mrms-v12-msqpe01h-p1-alaska'),
+    ('5fc5d74a-6684-4ffb-886a-663848ba22d9', 'marfc-rtmat-01h', 'marfc-rtmat-01h'),
+    ('e2228d8c-204a-4c7e-849b-a9a7e5c13eca', 'marfc-nbmt-03h', 'marfc-nbmt-03h'),
+    ('af651a3b-03ad-424d-8cf7-9ca7230309ed', 'marfc-nbmt-01h', 'marfc-nbmt-01h'),
+    ('7093dd22-2fa4-4172-b67d-5abc586e5eb6', 'marfc-fmat-06h', 'marfc-fmat-06h');
+
+-- suite
+INSERT INTO suite (id, name, slug, description) VALUES
+    ('839c527c-cd05-4d04-a533-bbf5c0b2a3b0', 'North Central River Forecast Center (NCRFC)', 'ncrfc', 'NCRFC Description'),
+    ('cc901b66-99fd-4671-9b68-a0bb8fc4223f', 'Middle Atlantic River Forecast Center (MARFC)', 'marfc', 'MARFC Description');
+
+-- product
+INSERT INTO product (id, slug, label, temporal_duration, temporal_resolution, dss_fpart, parameter_id, unit_id, description, suite_id) VALUES
+    ('0de4d4a8-2a18-4564-a14d-fb426c2046a8','ncrfc-mpe-01h','NCRFC QPE 01 Hr',3600,3600,'NCRFC QPE 01 HR','eb82d661-afe6-436a-b0df-2ab0b478a1af','e245d39f-3209-4e58-bfb7-4eae94b3f8dd','North Central River Forecast Center 01 hour QPE','839c527c-cd05-4d04-a533-bbf5c0b2a3b0'),
```

Comment:
1. I would remove Suite acronym from "label", the composite "Name" will include that already.
2. Use dashes in F-Part, don't include param or duration
3. "Label" shouldn't include duration